src/docfx/lib/markdown/MarkdownUtility.cs

```diff
@@ -86,7 +86,13 @@ public static (List<Error> errors, MarkdownDocument ast) Parse(string content, M
 
         private static MarkdownPipeline CreateMarkdownPipeline()
         {
-            var markdownContext = new MarkdownContext(GetToken, LogWarning, LogError, ReadFile);
+            var markdownContext = new MarkdownContext(
```

Comment:
```suggestion
            var markdownContext = new MarkdownContext(
                GetToken, LogInfo, LogSuggestion, LogWarning, LogError, ReadFile)
```

`LogInfo` can print everything `code, message, origin, line` to the console using the format defined in `ErrorLog.cs` line 240

Reply:
cool